resources/todomvc/vanilla-examples/javascript-web-components/src/styles/global.constructable.js

```diff
@@ -0,0 +1,5 @@
+const sheet = new CSSStyleSheet();
```

Comment:
I have a concern that this might diverge from global.css eventually. Would it be possible to fetch the content of global.css there, maybe at build time?

A simpler solution could be that you add a comment both here and there to make sure to update one when the other changes. It should also be easier if you use "`" template strings.

Reply:
the constructable css files (or any styles) will be updated once this pr lands: https://github.com/WebKit/Speedometer/pull/254

adjustments on how to generate them, should ideally happen there.

Reply:
re: diverge of global.css - the rollup plugin generates the constructable js files from the css files.
Once that pr is merged, we can install the styles as a local packages, similar to the news sites and that "should" keep things synced.

Reply:
sounds good to me